### ISSUE TITLE
Еще одна проблема с загрузкой

### ISSUE THREAD

**Issue by Webdesme** [avocode/avocode#1551]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.7`
Avocode UID: 31535
```
Error: ENOENT: no such file or directory, stat '/var/folders/gh/kx_l710j4s35tnp5qlmrvpjc0000gn/T/com.madebysource.avocode/avocode-sync-1500979901330/9FB0EADC-181C-4D20-8C95-2286BEA2D761/02F5FB64-F60E-494D-BECF-90D4E3B55598.png'
```

**Comment by Webdesme** [comment]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.7`
Avocode UID: 31535
```
Error: ENOENT: no such file or directory, stat '/var/folders/gh/kx_l710j4s35tnp5qlmrvpjc0000gn/T/com.madebysource.avocode/avocode-sync-1500980466563/C103C251-6507-4244-8B60-8304A21911A4/82E8B162-2AEB-43D0-9C09-23FDE65B08F2.png'
```

**Comment by connor-bean** [comment]
Can you tell us more about what you were doing when you received these errors?

**Comment by Webdesme** [comment]
I exported the layouts to the avacode, after the avacode showed that it was loading, this error will be output. Only with a specific project is the link to the sketch file https://www.dropbox.com/s/15rw1mnfmfj97et/9-19_new.sketch?dl=0

**Comment by jakubzitny** [comment]
Hi @Webdesme, thanks for the design, I'll look into it and get back to you.

**Comment by Webdesme** [comment]
Download here this artboard http://www.lavuix.ru/scr/170725150516.png

**Comment by Webdesme** [comment]
I deleted the symbols from the artboard and the layout booted

**Comment by Webdesme** [comment]
I deleted the symbols from the artboard and the layout booted

**Comment by jakubzitny** [comment]
Hi @Webdesme, there seems to be a problem with few layers in your file, for now, please sync the artboards separately or unlink the symbols, we'll try to fix this on our side for the next release.

Thanks for the file and for the info for now, we'll update you when the fix is ready 😉

**Comment by jakubzitny** [comment]
By the way, you can also use `Sync Without Symbols` from the Sketch plugin, you don't have to manually unlink the symbols.

![screen shot 2017-07-25 at 15 24 17](https://user-images.githubusercontent.com/3315662/28574158-5d863c06-714d-11e7-947a-4dbfb5b4daca.png)

**Comment by Webdesme** [comment]
![image](https://user-images.githubusercontent.com/12981695/28574778-951374a4-7157-11e7-8047-c60d9ec4d592.png)
I do not have such a menu item

**Comment by jakubzitny** [comment]
Allright, then there is probably another problem in your case. That is, the plugin update did not install to correct location. We found this problem yesterday in some versions of Sketch, that declare and use different plugin locations. This will be fixed for the next version as well.

Please download the latest plugin from [here](https://manager.avocode.com/download/sketch-plugin-new/mac/) and re-install the plugin manually.

**Comment by Webdesme** [comment]
Thanks for waiting for the new patch)

**Comment by jakubzitny** [comment]
Allright, I managet to come up with a fix to this problem. Please download another plugin update from [here](https://upx.cz/iYd) and let me know if it works. I was able to sync the individual artboard with related symbols already. I am now testing the rest of the file, it's pretty huge 😛

**Comment by Webdesme** [comment]
Very cool)! Everything is working! Thank you!

**Comment by jakubzitny** [comment]
Hi @Webdesme, we've fixed this problem in latest Avocode `2.14.8`, thanks for your cooperation. We will continue to improve the Sketch experience in Avocode even further in `2.15.0`.

Please update the app and let us know if you have other problems or issues :wink:
